string req
int32 markerId
---
string hintStatement
bool hintLoaded
bool completeHypo
